Traceback (most recent call last):
  File "/home/jordi/anaconda3/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/jordi/.local/lib/python3.9/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/jordi/.local/lib/python3.9/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/jordi/.local/lib/python3.9/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/home/jordi/anaconda3/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/home/jordi/.local/lib/python3.9/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/home/jordi/.local/lib/python3.9/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/jordi/.local/lib/python3.9/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
iris = iris.drop(index='Unnamed:0')
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mKeyError[0m                                  Traceback (most recent call last)
Input [0;32mIn [5][0m, in [0;36m<cell line: 1>[0;34m()[0m
[0;32m----> 1[0m iris [38;5;241m=[39m [43miris[49m[38;5;241;43m.[39;49m[43mdrop[49m[43m([49m[43mindex[49m[38;5;241;43m=[39;49m[38;5;124;43m'[39;49m[38;5;124;43mUnnamed:0[39;49m[38;5;124;43m'[39;49m[43m)[49m

File [0;32m~/anaconda3/lib/python3.9/site-packages/pandas/util/_decorators.py:311[0m, in [0;36mdeprecate_nonkeyword_arguments.<locals>.decorate.<locals>.wrapper[0;34m(*args, **kwargs)[0m
[1;32m    305[0m [38;5;28;01mif[39;00m [38;5;28mlen[39m(args) [38;5;241m>[39m num_allow_args:
[1;32m    306[0m     warnings[38;5;241m.[39mwarn(
[1;32m    307[0m         msg[38;5;241m.[39mformat(arguments[38;5;241m=[39marguments),
[1;32m    308[0m         [38;5;167;01mFutureWarning[39;00m,
[1;32m    309[0m         stacklevel[38;5;241m=[39mstacklevel,
[1;32m    310[0m     )
[0;32m--> 311[0m [38;5;28;01mreturn[39;00m [43mfunc[49m[43m([49m[38;5;241;43m*[39;49m[43margs[49m[43m,[49m[43m [49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[43mkwargs[49m[43m)[49m

File [0;32m~/anaconda3/lib/python3.9/site-packages/pandas/core/frame.py:4906[0m, in [0;36mDataFrame.drop[0;34m(self, labels, axis, index, columns, level, inplace, errors)[0m
[1;32m   4774[0m [38;5;129m@deprecate_nonkeyword_arguments[39m(version[38;5;241m=[39m[38;5;28;01mNone[39;00m, allowed_args[38;5;241m=[39m[[38;5;124m"[39m[38;5;124mself[39m[38;5;124m"[39m, [38;5;124m"[39m[38;5;124mlabels[39m[38;5;124m"[39m])
[1;32m   4775[0m [38;5;28;01mdef[39;00m [38;5;21mdrop[39m(
[1;32m   4776[0m     [38;5;28mself[39m,
[0;32m   (...)[0m
[1;32m   4783[0m     errors: [38;5;28mstr[39m [38;5;241m=[39m [38;5;124m"[39m[38;5;124mraise[39m[38;5;124m"[39m,
[1;32m   4784[0m ):
[1;32m   4785[0m     [38;5;124;03m"""[39;00m
[1;32m   4786[0m [38;5;124;03m    Drop specified labels from rows or columns.[39;00m
[1;32m   4787[0m 
[0;32m   (...)[0m
[1;32m   4904[0m [38;5;124;03m            weight  1.0     0.8[39;00m
[1;32m   4905[0m [38;5;124;03m    """[39;00m
[0;32m-> 4906[0m     [38;5;28;01mreturn[39;00m [38;5;28;43msuper[39;49m[43m([49m[43m)[49m[38;5;241;43m.[39;49m[43mdrop[49m[43m([49m
[1;32m   4907[0m [43m        [49m[43mlabels[49m[38;5;241;43m=[39;49m[43mlabels[49m[43m,[49m
[1;32m   4908[0m [43m        [49m[43maxis[49m[38;5;241;43m=[39;49m[43maxis[49m[43m,[49m
[1;32m   4909[0m [43m        [49m[43mindex[49m[38;5;241;43m=[39;49m[43mindex[49m[43m,[49m
[1;32m   4910[0m [43m        [49m[43mcolumns[49m[38;5;241;43m=[39;49m[43mcolumns[49m[43m,[49m
[1;32m   4911[0m [43m        [49m[43mlevel[49m[38;5;241;43m=[39;49m[43mlevel[49m[43m,[49m
[1;32m   4912[0m [43m        [49m[43minplace[49m[38;5;241;43m=[39;49m[43minplace[49m[43m,[49m
[1;32m   4913[0m [43m        [49m[43merrors[49m[38;5;241;43m=[39;49m[43merrors[49m[43m,[49m
[1;32m   4914[0m [43m    [49m[43m)[49m

File [0;32m~/anaconda3/lib/python3.9/site-packages/pandas/core/generic.py:4150[0m, in [0;36mNDFrame.drop[0;34m(self, labels, axis, index, columns, level, inplace, errors)[0m
[1;32m   4148[0m [38;5;28;01mfor[39;00m axis, labels [38;5;129;01min[39;00m axes[38;5;241m.[39mitems():
[1;32m   4149[0m     [38;5;28;01mif[39;00m labels [38;5;129;01mis[39;00m [38;5;129;01mnot[39;00m [38;5;28;01mNone[39;00m:
[0;32m-> 4150[0m         obj [38;5;241m=[39m [43mobj[49m[38;5;241;43m.[39;49m[43m_drop_axis[49m[43m([49m[43mlabels[49m[43m,[49m[43m [49m[43maxis[49m[43m,[49m[43m [49m[43mlevel[49m[38;5;241;43m=[39;49m[43mlevel[49m[43m,[49m[43m [49m[43merrors[49m[38;5;241;43m=[39;49m[43merrors[49m[43m)[49m
[1;32m   4152[0m [38;5;28;01mif[39;00m inplace:
[1;32m   4153[0m     [38;5;28mself[39m[38;5;241m.[39m_update_inplace(obj)

File [0;32m~/anaconda3/lib/python3.9/site-packages/pandas/core/generic.py:4185[0m, in [0;36mNDFrame._drop_axis[0;34m(self, labels, axis, level, errors)[0m
[1;32m   4183[0m         new_axis [38;5;241m=[39m axis[38;5;241m.[39mdrop(labels, level[38;5;241m=[39mlevel, errors[38;5;241m=[39merrors)
[1;32m   4184[0m     [38;5;28;01melse[39;00m:
[0;32m-> 4185[0m         new_axis [38;5;241m=[39m [43maxis[49m[38;5;241;43m.[39;49m[43mdrop[49m[43m([49m[43mlabels[49m[43m,[49m[43m [49m[43merrors[49m[38;5;241;43m=[39;49m[43merrors[49m[43m)[49m
[1;32m   4186[0m     result [38;5;241m=[39m [38;5;28mself[39m[38;5;241m.[39mreindex([38;5;241m*[39m[38;5;241m*[39m{axis_name: new_axis})
[1;32m   4188[0m [38;5;66;03m# Case for non-unique axis[39;00m
[1;32m   4189[0m [38;5;28;01melse[39;00m:

File [0;32m~/anaconda3/lib/python3.9/site-packages/pandas/core/indexes/base.py:6017[0m, in [0;36mIndex.drop[0;34m(self, labels, errors)[0m
[1;32m   6015[0m [38;5;28;01mif[39;00m mask[38;5;241m.[39many():
[1;32m   6016[0m     [38;5;28;01mif[39;00m errors [38;5;241m!=[39m [38;5;124m"[39m[38;5;124mignore[39m[38;5;124m"[39m:
[0;32m-> 6017[0m         [38;5;28;01mraise[39;00m [38;5;167;01mKeyError[39;00m([38;5;124mf[39m[38;5;124m"[39m[38;5;132;01m{[39;00mlabels[mask][38;5;132;01m}[39;00m[38;5;124m not found in axis[39m[38;5;124m"[39m)
[1;32m   6018[0m     indexer [38;5;241m=[39m indexer[[38;5;241m~[39mmask]
[1;32m   6019[0m [38;5;28;01mreturn[39;00m [38;5;28mself[39m[38;5;241m.[39mdelete(indexer)

[0;31mKeyError[0m: "['Unnamed:0'] not found in axis"
KeyError: "['Unnamed:0'] not found in axis"

